mesonbuild/linkers/detect.py

```diff
@@ -104,7 +104,7 @@ def guess_win_linker(env: 'Environment', compiler: T.List[str], comp_class: T.Ty
             prefix=comp_class.LINKER_PREFIX if use_linker_prefix else [],
             exelist=compiler, version=search_version(o), direct=invoked_directly)
     elif 'OPTLINK' in o:
-        # Opltink's stdout *may* beging with a \r character.
+        # Opltink's stdout *may* begin with a \r character.
```

Comment:
Also optltink -> optlink while we're at it. :D

Reply:
Good eye 522553a3b